src/stream-net/StreamClient.cs

```diff
@@ -71,11 +71,14 @@ private Uri GetBaseUrl()
                 case StreamApiLocation.USEast:
                     region = "us-east";
                     break;
-                case StreamApiLocation.USWest:

```

Comment:
we should keep these for backwards compatibility (ideally log a deprecation warning)